<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RXPathInput</class>
 <widget class="QWidget" name="RXPathInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0,2,0,3,0,2,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="nameInclude">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="extInclude">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>not</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="nameExclude">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="extExclude">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="matchCase">
     <property name="text">
      <string>case sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="notBinary">
     <property name="text">
      <string>not binary</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
